### ISSUE TITLE
Can't set the download path to my secondary harddisk (not showing on the settings menu)[Bug]

### ISSUE THREAD

**Issue by vigneshkannan522** [manga-download/hakuneko#3769]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Describe the bug**
I was using Hakuneko on windows a while back. And then I switched to Linux (POP_OS) and thankfully Hakuneko was working fine. But after I reinstalled POP_OS last week and installed hakuneko, 2 things happened:
1) The deb file version is now working. It just shows blank screen (I waited....but just blank screen).

2) So, I installed the Flatpak version, and it worked. But there was a problem. I can't set the download directory to my hard disk (2nd disk).

My laptop has 2 disks. 1) SSD and 2) HDD.

The first time, I don't remember if I was using the deb file version or the flatpak version. But I do remember setting the download path to the HDD since my SSD doesn't have a lot of space. But now after reinstalling the version 6.1.7, I can't set the download location to my hard disk. My HDD isn't even visible on the popup window to set the download location.

**Expected behavior**
All the disks should be visible on the popup window to select the download directory.

**Screenshots**
![Screenshot-20210801140735-1176x921](https://user-images.githubusercontent.com/64825629/127764829-9cf94a5d-e4d3-476a-af6c-9f45e0c65714.png)

![Screenshot-20210801141008-1176x921](https://user-images.githubusercontent.com/64825629/127764906-8d5c8ddb-fcc9-44a8-8f2b-21e94c5a5ca3.png)
In the above image, you can see that only the "computer" directory (the one on which the OS is installed) is shown. My hard disk is not visible.

**Context (please complete the following information):**
 - OS: Linux (POP_OS 21.04)
 - HakuNeko Version - 6.1.7

**Comment by Robonau** [comment]
is this not just a linux protecting your shit thing?
HN isn't what generates the folder selection box, your OS does

**Comment by vigneshkannan522** [comment]
I don't know. But it worked the last time. Just one week before, I was using it. I thought I might've been using the previous verison last time and since it installed the new one now, the dev might've changed it. I tried reinstalling HK, but it didn't work.
Do you have any suggestion?

**Comment by 27136773** [comment]
For blank screen see #13
https://hakuneko.download/docs/troubleshoot/#is-the-screen-blankblack-when-starting-hakuneko

The `Flatpak` package is maintained by the community and not part of this project. It might be possible that `Flatpak` is sandboxed and therefore has limited access to the file system...

**Comment by vigneshkannan522** [comment]
@ronny1982 Thanks for you reply. Then, I'll try installing the deb version and troubleshooting the blankscreen problem.

**Comment by vigneshkannan522** [comment]
@ronny1982 I solved the issue. I installed the deb version and like always it showed the blank screen. Then I went and manually patched it. Now it doesn't show the blank screen. The problem seems to be that there was no cache directory at all. There was no cache directory and tmp directory. So, I manually created those directories and pasted the extracted cache files from the zip file to the cache directory and restarted the client. Now it works fine. 

(But, why did it not create the cache directory? last time it worked, so it should've created it last time right? So, why did it not create it this time? I wonder....)

And it also solved the directory issue. Maybe you are right. The Flatpak version is sandboxed.

Anyway, once again thanks for your reply :)  :+1:
